CHANGELOG.md

```diff
@@ -18,18 +18,21 @@ NOTE: We are deprecating several collectors in this release.
 * [FEATURE] Add ZFS collector + review feedback from PRs 213 and 369 #410
```

Comment:
That's the literal string from the PR title.  Maybe we need to be more careful about reviewing PR titles.